velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -318,6 +318,8 @@ void VeloxToSubstraitPlanConvertor::constructFunctionMap() {
   functionMap_["sum"] = 5;
   functionMap_["mod"] = 6;
   functionMap_["eq"] = 7;
+  functionMap_["row_constructor"] = 8;
```

Comment:
I think we should keep this code now, because this PR add a UT test case named  VeloxSubstraitRoundTripPlanConverterTest.rowConstructor.

> auto plan = PlanBuilder()
                  .values({vectors})
                  .project({"row_constructor(c1, c2)"})
                  .planNode();